fastapi/readme/USAGE.rst

```diff
@@ -1228,6 +1224,119 @@ for all the endpoints. You can do the same for a specific app by checking the
 'app' field of the 'fastapi.endpoint' record before registering your custom error
 handler.
 
+FastAPI addons directory structure
+==================================
+
+When you develop a new addon to expose an api with fastapi, it's a good practice
+to follow the same directory structure and naming convention for the files
+related to the api. It will help you to easily find the files related to the api
+and it will help the other developers to understand your code.
+
+Here is the directory structure that we recommend. It's based on practices that
+are used in the python community when developing a fastapi app.
+
+.. code-block::
+
+  .
+  ├── api_x
```

Comment:
Actually, I liked to have `api` at the end. So the api modules are close to feature modules in alphabetical order.